lib/event.rb

```diff
@@ -0,0 +1,43 @@
+class Event
+  attr_accessor :name, :crafts, :attendees
+
+  def initialize(name, crafts, attendees)
+    @name = name
+    @crafts = crafts
+    @attendees = attendees
+  end
+
+  def attendee_names
+    names = @attendees.collect do |attendee|
+      attendee.name
+    end
+
+    names
+  end
+
+  def craft_with_most_supplies
+    supply_count = {}
```

Comment:
Can you think of another enumerable that could be used to get the max? HINT: `.max_by`